tests/all/js_globals/Math.rs

```diff
@@ -78,6 +78,7 @@ fn acosh() {
             export function test() {
                 assert.equal(wasm.acosh(1), 0);
                 assert.equal(wasm.acosh(2), Math.acosh(2));
+                assert.equal(wasm.acosh(2), Math.acosh(2));
```

Comment:
This seems to be a line added by mistake?

Reply:
I didn't even work on `acosh` .. weird.